services/agents/agents.go

```diff
@@ -0,0 +1,20 @@
+package agents
+
+import (
+	"github.com/AlekSi/pointer"
+
+	"github.com/percona/pmm-managed/models"
+)
+
+type debugValue bool
+
+var enableDebug = debugValue(true)
+var disableDebug = debugValue(false)
+
+func redactKeywords(s *models.Agent, debug debugValue) []string {
```

Comment:
Why this function returns an array if as maximum, it will have only one element?

Reply:
because in future it can be changed

Reply:
Added `AWSSecretKey` to the list.